public/app/plugins/datasource/alertmanager/ConfigEditor.tsx

```diff
@@ -73,6 +73,11 @@ export const ConfigEditor = (props: Props) => {
             />
           </InlineField>
         </div>
+        {options.jsonData.handleGrafanaManagedAlerts && (
+          <span>
+            Make sure to enable the alert forwarding on the <a href="/alerting/admin">admin page.</a>
```

Comment:
I'm not sure if this is the best way to link internally.

Reply:
Use the `<Link to='/alerting/admin'></Link>` component from `react-router-dom` :)